java/README.md

```diff
@@ -43,6 +44,13 @@ make clean
 make android
 ```
 
+## Building on Windows
+
+```powershell
+# Important: Ensure Java 1.8 is used when compiling
+$env:JAVA_HOME="C:\Program Files\Java\jdk-1.8"
+```
```

Comment:
These aren't actually build instructions, running this won't result in a build. This seems more like a note. If we're adding a `Building on Windows` section, it should include the steps to build on Windows.

Reply:
@smcintyre-r7  I try to add some more description, and I check with this steps, we can build on windows successfully

But I am not good in English, so if you have any idea to make better sentences, I am very welcome :)